doc/DEPLOYMENT.md

```diff
@@ -485,3 +485,27 @@ Final steps
     * `local_templates/support_channels_en.html`
     * `local_templates/support_channels_fi.html`
 
+Integration with Student Information System
+-------------------------------------------
+
+A+ can be integrated with the organisation's Student Information System (SIS,
+a system that manages information on courses and its participants).
+For example, Aalto University, along with a few other Finnish universities
+use "SISU" for this purpose. The integration with the organisation-specific
+SIS system is done using a SIS plugin. For example, Aalto's SISU plugin
+can be found in [GitHub](https://github.com/apluslms/aalto-sisu).
```

Comment:
We should add a link to the plugin interface class here. Mentioning plugins and the interface without linking the definition of the interface is confusing.